storage/spider/mysql-test/spider/t/spider_udf_in_where_conds.test

```diff
@@ -0,0 +1,138 @@
+--disable_warnings
```

Comment:
I think the name of the test file `spider_udf_in_where_conds.test` can be simplified. It is obvious that the test is for spider. And, I want to make it clear that the test is about condition pushdown. So, how about `udf_pushdown.test`? 
 Same as the result file.